var/spack/repos/builtin/packages/alps/package.py

```diff
@@ -0,0 +1,96 @@
+# Copyright Spack Project Developers. See COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class Alps(CMakePackage):
+    """Algorithms for Physics Simulations
+
+    Tags: Condensed Matter Physics, Computational Physics
```

Comment:
What are these tags for? How do they relate to the `tags` class level attribute?